topology/sof/pipe-volume-playback.m4

```diff
@@ -67,5 +67,5 @@ indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Passthrough Playback PCM_I
 # PCM Configuration
 
 #
-PCM_CAPABILITIES(Passthrough Playback PCM_ID, `S32_LE,S24_LE,S16_LE', 48000, 48000, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
+PCM_CAPABILITIES(Passthrough Playback PCM_ID, `S32_LE,S24_LE,S16_LE', 48000, 48000, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 7680, 30720)
 
```

Comment:
better to make the buffer range larger than [20ms, 80ms], e.g. make it to be [4ms, 2s], then it can meet almost all kinds of requirement.

Reply:
@keyonjie I cant make the min buffer size less than 20ms as youtube requires a minimum of 20ms and it fails to play with anything less